src/components/staking/PageValidator.vue

```diff
@@ -309,6 +312,7 @@ export default {
         }
       },
       update: result => {
+        
```

Comment:
this new line is not necessary

how is it that these types of lint things are getting through?